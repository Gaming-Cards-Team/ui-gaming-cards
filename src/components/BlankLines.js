import React from "react";

const BlankLines = ({numberLines = 3}) => {
   
  const lines = Array.from({length: numberLines}, (v, i) => i)
  return (
    <> 
      {lines.map((item) => (<br key={item}/>))}
    </>
  );
};

export default BlankLines;
