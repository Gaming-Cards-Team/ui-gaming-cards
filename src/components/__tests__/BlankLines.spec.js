import React from 'react';
import BlankLines from '../BlankLines';
import renderer from 'react-test-renderer';

describe('Test Banner component', () => {

    it('should validate the component', () => {
        const component = renderer.create(<BlankLines />).toJSON();

        expect(component).toMatchSnapshot();
    });
});