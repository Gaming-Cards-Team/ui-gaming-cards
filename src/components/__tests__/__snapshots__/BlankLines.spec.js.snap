// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Banner component should validate the component 1`] = `
Array [
  <br />,
  <br />,
  <br />,
]
`;
